Dockerfile

```
@@ -21,7 +21,7 @@ RUN        mkdir -p /data/{warc,cdxj,ipfs}
 
 # Download and install IPFS
 ENV        IPFS_PATH=/data/ipfs
-ARG        IPFS_VERSION=v0.30.0
+ARG        IPFS_VERSION=v0.31.0
 ARG        BUILDARCH
 RUN        cd /tmp \
            && wget -q https://dist.ipfs.io/kubo/${IPFS_VERSION}/kubo_${IPFS_VERSION}_linux-$BUILDARCH.tar.gz \
```

nit: switch to .tech to avoid redirect
```suggestion
           && wget -q https://dist.ipfs.tech/kubo/${IPFS_VERSION}/kubo_${IPFS_VERSION}_linux-$BUILDARCH.tar.gz \
```

---

Thank you for the heads-up, @lidel! I was not aware of the redirect in-place but have followed the news on the .io TLD, so figure some changes was forthcoming.